README.md

```diff
@@ -45,7 +45,7 @@ A more in-depth explaination of the concepts below can be found [here](./game-ru
 
 ### Step 1
 Let's start off by running your very first match:
-(Because we are going to run Java files, please make sure you have [java installed](http://www.oracle.com/technetwork/pt/java/javase/downloads/jdk8-downloads-2133151.html))
+(Because we are going to run Javascript files, please make sure you have [Nodejs installed](https://nodejs.org/en/download/)
```

Comment:
We need java and node right? The bot is js but the runner is java...

Reply:
Yes the javascript bot needs NodeJs to run
The runner needs Java but that is handled in it's own readme

Reply:
Fair enough. But this is the main readme - specifically the starter pack section . Which is why I asked.